internal/nostr/nostr.go

```diff
@@ -719,8 +719,8 @@ func (svc *Service) startSubscription(ctx context.Context, subscription *Subscri
 			relay.Close()
 		}
 		if ctx.Err() != nil {
-			svc.Logger.WithFields(logrus.Fields{
-				"subscription_id": subscription.ID,
+			svc.Logger.WithError(ctx.Err()).WithFields(logrus.Fields{
+				"subscription_id": subscription.Uuid,
 				"relay_url":       subscription.RelayUrl,
 			}).Debug("Context canceled, stopping subscription")
```

Comment:
Do we really want to change this to error level?

Reply:
I think so - I don't think we want to receive context cancelled errors, do we? I guess this means a hub is offline and we cannot do anything, but it's something that is good to know. Maybe getalby.com eventually should stop trying to request to offline hubs CC @bumi ?

Reply:
True, I was thinking from one-off susbcriptions pov where this might be hit because of timeout, thereby causing bloat, will address this too in the one-off separation PR :+1:

Reply:
Leaving this as Error level for now (the separated one-off subscription function will have debug log)

Reply:
@im-adithya 
> where this might be hit because of timeout
I still think that we should know this is happening and not just hide it? I still think we need to solve the root problem.